WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaInserterHelper.swift

```diff
@@ -164,6 +178,28 @@ class GutenbergMediaInserterHelper: NSObject {
             gutenberg.mediaUploadUpdate(id: mediaUploadID, state: .uploading, progress: Float(value), url: nil, serverID: nil)
         }
     }
+
+    func fetchVideoPressUrl(for media: Media, completion: @escaping ( Result<(videoURL: URL, posterURL: URL?)> ) -> Void) {
```

Comment:
I think the name should be `fetchVideoPressURL`

Reply:
that methods implementation has changed so I updated its name as `fetchRemoteURL`